<launch>
    <param name="ec_cluster_tolerance" value="0.01" />
    <param name="ec_min_cluster_size" value="968" />
    <param name="ec_max_cluster_size" value="1262" />
    <param name="distance_above_plane" value="0.01" />
    <param name="crop_min_x" value="-0.5" />
    <param name="crop_max_z" value="1.5" />
    <param name="crop_max_x" value="1.05" />
    <param name="crop_min_z" value="0.7" />
    <node name="$(anon perception_demo)" pkg="perception" type="point_cloud_demo" args="/home/team1/data/labels cloud_in:=/head_camera/depth_registered/points"/>
    <node name="$(anon gripper_teleop)" pkg="applications" type="gripper_teleop.py"/>
</launch>
